collect_app/src/main/java/org/odk/collect/android/widgets/utilities/RangeWidgetUtils.java

```diff
@@ -0,0 +1,118 @@
+package org.odk.collect.android.widgets.utilities;
+
+import android.annotation.SuppressLint;
+import android.app.Activity;
+import android.content.Context;
+import android.view.MotionEvent;
+import android.view.View;
+import android.widget.TextView;
+
+import org.javarosa.core.model.RangeQuestion;
+import org.javarosa.form.api.FormEntryPrompt;
+import org.odk.collect.android.R;
+import org.odk.collect.android.databinding.RangeWidgetHorizontalBinding;
+import org.odk.collect.android.databinding.RangeWidgetVerticalBinding;
+import org.odk.collect.android.utilities.ToastUtils;
+import org.odk.collect.android.views.CustomRangeSlider;
+
+import java.math.BigDecimal;
+
+public class RangeWidgetUtils {
+    private static final String VERTICAL_APPEARANCE = "vertical";
+
+    private RangeWidgetUtils() {
+    }
+
+    public static Object[] setUpLayoutElements(Context context, FormEntryPrompt prompt) {
+        View answerView;
+        CustomRangeSlider slider;
+        TextView currentValue;
+        TextView minValue;
+        TextView maxValue;
+
+        String appearance = prompt.getQuestion().getAppearanceAttr();
+
+        if (appearance != null && appearance.contains(VERTICAL_APPEARANCE)) {
```

Comment:
It looks like there aren't any tests to check `VERTICAL_APPEARANCE` works.

Reply:
Right @seadowg! I was having some difficulty in writing the tests for the `VERTICAL_APPEARANCE`. I think checking the rotation of the slider would be fine here.

Reply:
Yeah that sounds good enough! Maybe just check in both cases all the details are rendered and that the `Slider` is rotated.